x-pack/plugins/files/server/routes/file_kind/integration_tests/file_kind_http.test.ts

```diff
@@ -104,12 +104,12 @@ describe('File kind HTTP API', () => {
   test('create', async () => {
     expect(await createFile()).toEqual({
       id: expect.any(String),
-      created_at: expect.any(String),
-      updated_at: expect.any(String),
+      created: expect.any(String),
+      updated: expect.any(String),
       name: 'myFile',
-      file_kind: fileKind,
+      fileKind,
       status: 'AWAITING_UPLOAD',
-      mime: 'image/png',
+      mimeType: 'image/png',
```

Comment:
Shall this return as per ECS, e.g. `Status` instead of `status`? Or the idea here is to have them in JS-friendly format?

Reply:
Yeah exactly, the thinking was to expose JS friendly version of the JSON attributes from our endpoints and to other Kibana code interacting with `files`. Happy to revisit this assumption though.